thrust/thrust/mr/allocator.h

```diff
@@ -105,7 +105,7 @@ class allocator : private validator<MR>
   _CCCL_EXEC_CHECK_DISABLE
   _CCCL_HOST_DEVICE size_type max_size() const
   {
-    return (std::numeric_limits<size_type>::max)() / sizeof(T);
+    return (::cuda::std::numeric_limits<size_type>::max)() / sizeof(T);
```

Comment:
```suggestion
    return ::cuda::std::numeric_limits<size_type>::max() / sizeof(T);
```

There may be more of those

Reply:
That is actually a defense against windows having `max` as a function like macro